src/Compilers/CSharp/Test/Semantic/Semantics/InheritanceBindingTests.cs

```diff
@@ -9794,5 +9794,29 @@ class C2 : I
 ";
             var comp = CreateCompilation(source).VerifyDiagnostics();
         }
+
+        [Fact]
+        [WorkItem(63490, "https://github.com/dotnet/roslyn/issues/63490")]
+        public void MultipleBasesWithObliviousDifferencesAndInterfaces()
```

Comment:
>MultipleBasesWithObliviousDifferencesAndInterfaces [](http://example.com/codeflow?start=20&length=50)

Consider testing with different order for partial declarations to make sure the behavior is not order dependent